phoenix-core/src/it/java/org/apache/phoenix/end2end/TTLAsPhoenixTTLIT.java

```diff
@@ -370,4 +455,62 @@ private String createViewOnViewWithTTL(Connection conn, String parentViewName,
         return childView;
     }
 
+
```

Comment:
You may need to add tests for resetting TTL (TTL=NONE) at a given level and then setting (ALTER) to new TTL at a different level.

Currently Alter table  TTL=NONE sets it to 0 and thus cannot set a new TTL at a different level.